### PR TITLE
[EDU-5669] feature: add and publish Savings Plan Terms and Conditions

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-5669]

### Changes

Add and publish the Terms and Conditions of the Savings Plan files EN/PT.
Include the link in the Agreements page EN/PT.

### Additional links

n/a.

[EDU-5669]: https://aziontech.atlassian.net/browse/EDU-5669?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by isidrohernandezazion — APPROVED** [review]

**Review by guiafonso-ol — APPROVED** [review]

File: src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+3.1. In the event of non-compliance with any clause of the Customer Agreement, as well as any other contract or term agreed upon between the parties, including the Service Order, the Customer shall be subject to the disregard of the Discount on the On Demand Price and retroactive billing from the beginning of the Term of Validity of the amounts recalculated as On Demand consumption.
+
+3.2. The Customer is aware and agrees that the Products and Services will be subject to the On Demand Prices, billed monthly, when: (a) their consumption exceeds the Purchase Commitment and any credits acquired or received; (b) using Products and Services not eligible for the Discount on the On Demand Price or that are not specified in the Service Order; or (c) after the Term of Validity, there is no automatic renewal or signing of a new Service Order.
+
+3.3. In the event of a conflict or divergence between the conditions defined herein and in the Service Order, the provisions of the Service Order shall prevail.
+
+## 4. Definitions
+
+4.1 The following are added to the Terms and Definitions of the Customer Agreement:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx:41]: A numer

Comment (MarianaAguilera) [inline src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx:41]:
```suggestion
4.1. The following are added to the Terms and Definitions of the Customer Agreement:
```

File: src/content/docs/pt-br/pages/contratos/savings-plan/savings-plan.mdx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+3.1. No caso de descumprimento de qualquer cláusula do Contrato de Cliente, bem como de qualquer outro contrato ou termo acordado entre as partes, inclusive da Ordem de Serviço, o Cliente estará sujeito à desconsideração do Desconto Sobre o Preço On Demand e cobrança retroativa ao início do Período de Vigência dos valores recalculados como consumo On Demand.
+
+3.2. O Cliente está ciente e concorda que os Produtos e Serviços estarão sujeitos aos Preços On Demand, faturados mensalmente, quando: (a) seu consumo superar o Compromisso de Compra e eventuais créditos adquiridos ou recebidos; (b) usar Produtos e Serviços não elegíveis ao Desconto Sobre o Preço On Demand ou que não estejam dispostos na Ordem de Serviço; ou (c) após o Período de Vigência, não houver renovação automática ou assinatura de uma nova Ordem de Serviço.
+
+3.3. Em caso de conflito ou divergência entre as condições definidas aqui e na Ordem de Serviço, as previsões da Ordem de Serviço deverão prevalecer.
+
+## 4. Definições
+
+4.1 Acrescentam-se aos Termos e Definições do Contrato de Cliente:
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/contratos/savings-plan/savings-plan.mdx:41]:
```suggestion
4.1. Acrescentam-se aos Termos e Definições do Contrato de Cliente:
```

**Review by mrmessa — CHANGES_REQUESTED** [review]
@MarianaAguilera, attached to the issue on Jira, we had a review of the legal to English text.

**Review by mrmessa — CHANGES_REQUESTED** [review]

File: src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+---
+title: Savings Plan Terms and Conditions
+description: Terms and Conditions of the Savings Plan
+meta_tags: Terms and Conditions of the Savings Plan, Azion 
+namespace: docs_agreements_terms_and_conditions_savings_plan
+permalink: /documentation/agreements/terms-and-conditions-savings-plan/
+---
+
+Savings Plan is one of the ways of contracting Azion Products and Services, subject to the terms of the [Customer Agreement](https://www.azion.com/en/documentation/agreements/customer-agreement/) and other related documents. In this format, the Customer commits to pay a financial amount during the Effective Term (the "Purchase Commitment") and can enjoy discounts on On Demand Prices published on the subscription date, in accordance with their Service Order ("On Demand Price Discount"), except on third-party products listed on the Azion Marketplace.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx:9]:
```suggestion
Savings Plan is one of the ways of contracting Azion Products and Services, subject to the terms of the [Customer Agreement](/en/documentation/agreements/customer-agreement/) and other related documents. In this format, the Customer commits to pay a financial amount during the Effective Term (the "Purchase Commitment") and can enjoy discounts on On Demand Prices published on the subscription date, in accordance with their Service Order ("On Demand Price Discount"), except on third-party products listed on the Azion Marketplace.
```

File: src/content/docs/en/pages/agreements/2021-01-14-index.mdx [hunk 4]
```diff
@@ -12,3 +12,4 @@ permalink: /documentation/agreements/
 - [Acceptable Use Policy](/en/documentation/agreements/acceptable-use-policy/)
 - [Privacy Policy](/en/documentation/agreements/privacy-policy/)
 - [Marketplace Agreement](/en/documentation/agreements/marketplace/marketplace-agreement/)
+- [Terms and Conditions of the Savings Plan](/en/documentation/agreements/terms-and-conditions-savings-plan/)
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/agreements/2021-01-14-index.mdx:15]:
```suggestion
- [Savings Plan Terms and Conditions](/en/documentation/agreements/savings-plan-terms-and-conditions/)
```

File: src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+---
+title: Savings Plan Terms and Conditions
+description: Terms and Conditions of the Savings Plan
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/agreements/savings-plan/savings-plan.mdx:3]:
```suggestion
description: Terms and Conditions related to the Savings Plan
```

**Review by mrmessa — APPROVED** [review]

**Pull request merged**